monitor/cli.py

```diff
@@ -193,15 +194,21 @@ def monitor(
     type=str,
     help="The file path where Elementary's report will be saved."
 )
+@click.option(
+    '--disable-elementary-passed-tests-samples',
+    type=bool,
+    default=False,
+    help="If true, elementary won't query sample data for elementary passed tests (can improve report run time)."
```

Comment:
cosmetics - 'If set to true elementary report won't show data metrics for passed tests (this can improve report creation time)'